src/Microsoft.ML.Transforms/Text/NgramTransform.cs

```diff
@@ -253,7 +253,7 @@ private static SequencePool[] Train(IHostEnvironment env, NgramExtractingEstimat
                     // Note: GetNgramIdFinderAdd will control how many ngrams of a specific length will
                     // be added (using lims[iinfo]), therefore we set slotLim to the maximum
                     helpers[iinfo] = new NgramBufferBuilder(ngramLength, skipLength, Utils.ArrayMaxSize,
-                        GetNgramIdFinderAdd(env, counts[iinfo], columns[iinfo].Limits, ngramMaps[iinfo], transformInfos[iinfo].RequireIdf));
+                        GetNgramIdFinderAdd(env, counts[iinfo], columns[iinfo].MaximumTermCounts, ngramMaps[iinfo], transformInfos[iinfo].RequireIdf));
```

Comment:
>MaximumTermCounts [](start = 79, length = 17)

shoult Term be plural here: 'MaximumTermsCounts'

Reply:
Ok.

---
In reply to: [264022560](https://github.com/dotnet/machinelearning/pull/2898#discussion_r264022560) [](ancestors = 264022560)